### PR TITLE
Jcs/admin

### PR DESCRIPTION
**Pull request by jSylvestre**

### CODE REVIEW

**Review by sprucely — APPROVED** [review]
Is there a VS Code extension that can do this formatting, or was it just VS?

File: Hippo.Core/Services/AccessConfig.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        public static string[] GetRoles(string accessCode)
+        {
+            return accessCode switch
+            {
+                // System can access anything
+                AccessCodes.SystemAccess => new[] { RoleCodes.SystemRole },
+                // FieldManager can access anything restricted to FieldManager, Supervisor, Worker or PI roles
+                AccessCodes.AdminAccess => new[] { RoleCodes.AdminRole },
+                // Supervisor can access anything restricted to Supervisor, Worker or PI roles
+                _ => throw new ArgumentException($"{nameof(accessCode)} is not a valid {nameof(AccessCodes)} constant")
```
Comment (sprucely) [inline Hippo.Core/Services/AccessConfig.cs:17]: You may want to remove comments specific to Harvest.

File: Hippo.Web/Handlers/VerifyRoleAccessHandler.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            var userIamId = context.User.Claims.SingleOrDefault(c => c.Type == UserService.IamIdClaimType)?.Value;
+            var kerbId = context.User.Claims.SingleOrDefault(a => a.Type == ClaimTypes.NameIdentifier)?.Value;
+
+
+            if (string.IsNullOrWhiteSpace(userIamId))
+            {
+                return;
+            }
+
+            var systemUsers = new[] { "jsylvest", "postit", "cydoval", "sweber" };
```
Comment (sprucely) [inline Hippo.Web/Handlers/VerifyRoleAccessHandler.cs:32]: I forget. Did we decide that creating a `Roles` table would be overkill? At least for the first version?

**Review by sprucely — APPROVED** [review]
Looks fine to me. (Once the harvest comments are removed)

File: Hippo.Web/Handlers/VerifyRoleAccessHandler.cs (hunk 2 above)
Comment (jSylvestre) [reply Hippo.Web/Handlers/VerifyRoleAccessHandler.cs:32]: @sprucely I think that was what Scott said.

**Pull request merged**